packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx

```diff
@@ -95,45 +110,71 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
     paddingVertical: spacing(1),
     paddingHorizontal: spacing(2)
   },
+  ctaContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: spacing(2),
+    backgroundColor: palette.neutralLight9,
+    borderWidth: 1,
+    borderColor: palette.neutralLight7,
+    borderRadius: spacing(1),
+    paddingHorizontal: spacing(4),
+    paddingVertical: spacing(2)
+  },
   dim: {
     opacity: 0.5
   }
 }))
 
+const ctaToTextMap = {
+  [ChatPermissionAction.TIP]: messages.ctaTip,
+  [ChatPermissionAction.UNBLOCK]: messages.ctaBlock,
+  [ChatPermissionAction.NONE]: messages.ctaNone
+}
+
 type ChatUserListItemProps = {
   user: User
 }
 
 export const ChatUserListItem = ({ user }: ChatUserListItemProps) => {
   const styles = useStyles()
-  const palette = useThemeColors()
   const dispatch = useDispatch()
   const currentUserId = useSelector(getUserId)
-  const { canCreateChat } = useSelector((state) =>
+  const { callToAction, canCreateChat } = useSelector((state) =>
     getCanCreateChat(state, { userId: user.user_id })
   )
 
-  const handlePress = useCallback(
-    (user) => {
-      dispatch(createChat({ userIds: [user.user_id] }))
-    },
-    [dispatch]
-  )
+  const handlePress = useCallback(() => {
+    dispatch(createChat({ userIds: [user.user_id] }))
+  }, [dispatch, user.user_id])
+
+  const handleNotPermittedPress = useCallback(() => {
+    if (user.user_id) {
+      dispatch(
+        setVisibility({
+          drawer: 'InboxUnavailable',
+          visible: true,
+          data: { userId: user.user_id }
+        })
+      )
+    }
+  }, [dispatch, user.user_id])
 
   if (currentUserId === user.user_id) {
     return null
   }
 
   return (
     <TouchableOpacity
-      onPress={() => handlePress(user)}
-      disabled={!canCreateChat}
+      onPress={canCreateChat ? handlePress : handleNotPermittedPress}
     >
       <View style={styles.border}>
-        <View
-          style={[styles.userContainer, !canCreateChat ? styles.dim : null]}
-        >
-          <ProfilePicture profile={user} style={styles.profilePicture} />
+        <View style={styles.userContainer}>
+          <ProfilePicture
+            profile={user}
+            style={[styles.profilePicture, !canCreateChat ? styles.dim : null]}
```

Comment:
Just double checking - do we still dim this?

Reply:
We dim just the profile picture (according to figma)